File: moviejava/app/src/main/java/com/example/moviejava/model/casts/CastsFilm.java
package com.example.moviejava.model.casts;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;


public class CastsFilm {

    @SerializedName("id")
    public int id;
    @SerializedName("cast")
    public ArrayList<Cast> cast;
    @SerializedName("crew")
    public ArrayList<Crew> crew;

}

File: movie/app/src/main/java/com/example/movie/model/casts/CastsFilm.kt
package com.example.movie.model.casts

import com.google.gson.annotations.SerializedName


data class CastsFilm
    (

    @SerializedName("id")
    var id: Int? = null,
    @SerializedName("cast")
    var cast: ArrayList<Cast> = arrayListOf(),
    @SerializedName("crew")
    var crew: ArrayList<Crew> = arrayListOf()

)